subsys/bluetooth/services/alexa_gadgets/Kconfig.gadgets

```diff
@@ -1,12 +1,24 @@
 #
-# Copyright (c) 2020 Nordic Semiconductor
+# Copyright (c) 2020 Nordic Semiconductor ASA
 #
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-menu "Alexa Gadget Profile"
+config BT_GADGETS
+	bool "Gadgets service"
```

Comment:
"Alexa Gadgets service

Reply:
also I think `ALEXA` would be good to add to Kconfig symbols.
`GADGETS` is a bit too generic, imo.

Reply:
Updating as suggested